content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -282,12 +282,25 @@ must be able to receive OTLP over `grpc`. Therefore, the example uses
 
 #### Excluding auto-instrumentation {#js-excluding-auto-instrumentation}
 
-By default, the Node.js auto-instrumentation ships with
-[many instrumentation libraries](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-instrumentations-node/README.md#supported-instrumentations).
-At the moment, there is no way to opt-in to only specific packages or disable
-specific packages. If you don't want to use a package included by the default
-image you must either supply your own image that includes only the packages you
-want or use manual instrumentation.
+By default, the Node.js auto-instrumentation has all the
+[supported instrumentation libraries](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-instrumentations-node/README.md#supported-instrumentations)
```

Comment:
I removed this link, to avoid duplicating information present in [Node.js auto-instrumentation documentation](/docs/languages/js/automatic/configuration/), that is linked bellow